### DISCUSSION TITLE
Anyway to get all the markets available on an exchange?

### DISCUSSION THREAD

**Discussion by kaidatavis** [altangent/ccxws#288] [Q&A]
In CCXT it is possible to get the list of all the markets available from an exchange using `exchange.symbols` or `exchange.markets`. Is there a way to do something similar in CCXWS? 

Currently I need to find out what markets are available from the exchange site and then add them manually in my code. 

Also, is the way market specified in CCXWS (such as 'BTCUSDT') the same as the 'symbols' in CCXT (such as 'BTC/USDT)? If there is a way to convert CCXT 'symbol' to CCXWS market ID, I can use CCXT to get the list of interested markets and then retrieve the data with CCXWS.

**Comment by bmancini55** [comment]
Unfortunately, CCXWS does not currently support obtaining markets.  It could certainly be in scope for the library to obtain them via a REST request, however I don't think we would every attempt to normalize symbols across exchanges.

That said, CCXWS was designed to be compatible with CCXT markets, so you can use CCXT to obtain markets first. The primary benefit of CCXT is that the symbols will be normalized across exchanges.
